switcheroo/ssh/metric_constants.py

```diff
@@ -0,0 +1,6 @@
+# pylint: disable=too-few-public-methods
```

Comment:
On the structure - maybe have this class inside the /ssh/__init__.py folder. Then we can do from switcheroo.ssh import MetricConstants